src/components/managerlib/pdfwatcher.ts

```diff
@@ -5,12 +5,19 @@ import type {Extension} from '../../main'
 
 export class PdfWatcher {
     private readonly extension: Extension
-    private readonly pdfsWatched = new Set<string>()
+    private readonly watchedPdfLocalPaths = new Set<string>()
     private readonly pdfWatcher: chokidar.FSWatcher
+    private readonly watchedPdfVirtualUris = new Set<string>()
+    private readonly ignoredPdfUris = new Set<string>()
 
     constructor(extension: Extension) {
         this.extension = extension
         this.pdfWatcher = this.initiatePdfWatcher()
+        this.initiateVirtualUriWatcher()
```

Comment:
This function returns a file system watcher but the returned values is discarded. Not sure this is an issue but it looks wieird.

Reply:
It is intended. At the moment, we don't have to store the returned value as a property of `PdfWatcher`. If it becomes necessary in the future, we will store the returned value.